instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/datasource/OpenTelemetryDataSourceBuilder.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jdbc.datasource;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.jdbc.internal.JdbcInstrumenterFactory;
+import javax.sql.DataSource;
+
+public class OpenTelemetryDataSourceBuilder {
+
+  private final DataSource delegate;
+
+  private final OpenTelemetry openTelemetry;
+
+  private boolean dataSourceInstrumenterEnabled = true;
+
+  private boolean statementInstrumenterEnabled = true;
+
+  private boolean statementSanitizationEnabled = true;
+
+  OpenTelemetryDataSourceBuilder(DataSource delegate, OpenTelemetry openTelemetry) {
```

Comment:
Could it be private?

Reply:
I indented this to be `public` however I can make it `private` if I provide a public static method in the class to instantiate the builder. I have not preference here.